Exiled.Events/Handlers/Item.cs

```diff
@@ -9,44 +9,42 @@ namespace Exiled.Events.Handlers
 {
     using Exiled.Events.EventArgs.Item;
 
-    using Extensions;
-
-    using static Events;
+    using Exiled.Events.Features;
 
     /// <summary>
     ///     Item related events.
     /// </summary>
     public static class Item
```

Comment:
The doc summaries for all of the events should be changed. Either by omitting the `Gets or sets` text, or by changing it to something that makes sense.
This would be a good example of selectively disabling pragma warnings, as the `gets or sets` text is pretty much irrelevant here.